src/main/java/com/conveyal/gtfs/loader/JdbcTableWriter.java

```diff
@@ -492,6 +570,41 @@ private void updateChildTable(ArrayNode subEntities, Integer id, boolean isCreat
         }
     }
 
+    /**
+     * Updates the stop times that reference the specified pattern stop.
+     * @param patternStop the pattern stop for which to update stop times
+     * @param previousTravelTime the travel time accumulated up to the previous stop_time's departure time (or the
+     *                           previous pattern stop's dwell time)
+     * @return the travel and dwell time added by this pattern stop
+     * @throws SQLException
+     */
+    private int updateStopTimesForPatternStop(ObjectNode patternStop, int previousTravelTime) throws SQLException {
+        String sql = String.format(
+            "update %s.stop_times st set arrival_time = ?, departure_time = ? from %s.trips t " +
+                "where st.trip_id = t.trip_id AND t.%s = ? AND st.%s = ?",
+            tablePrefix,
+            tablePrefix,
+            "pattern_id",
+            "stop_sequence"
+        );
+        // Prepare the statement and set statement parameters
+        PreparedStatement statement = connection.prepareStatement(sql);
+        int oneBasedIndex = 1;
+        int travelTime = patternStop.get("default_travel_time").asInt();
+        int arrivalTime = previousTravelTime + travelTime;
+        statement.setInt(oneBasedIndex++, arrivalTime);
+        int dwellTime = patternStop.get("default_dwell_time").asInt();
+        statement.setInt(oneBasedIndex++, arrivalTime + dwellTime);
+        // Set "where clause" with value for pattern_id and stop_sequence
+        statement.setString(oneBasedIndex++, patternStop.get("pattern_id").asText());
+        statement.setInt(oneBasedIndex++, patternStop.get("stop_sequence").asInt());
```

Comment:
Using a variable for the prepared statement variable index seems a little too fancy.  Since the variables aren't dynamically created and the length of the parameters isn't varying, I think it may be best to simply hardcode the field position number.

Reply:
It seems more maintainable and less error-prone to me to use an incrementing variable. As long as the you're setting the parameters in the correct order, adding a new parameter is simply a matter of inserting it in the right spot and using the same `oneBasedIndex`. If we were to use hardcoded numbers, updating the statement could involve updating multiple numbers one by one if the new parameter is placed anywhere except the end.

Reply:
ok, I don't have a super strong opinion on this, so go ahead.